tests/github/immersive-web/webvr/spec/1.1/index.html

```diff
@@ -572,6 +572,8 @@ <h2 class="no-num no-toc no-ref heading settled" id="subtitle"><span class="cont
     <dl>
      <dt>This version:
      <dd><a class="u-url" href="https://immersive-web.github.io/webvr/">https://immersive-web.github.io/webvr/</a>
+     <dt>Feedback:
+     <dd><span><a href="mailto:public-webvr@mozilla.org?subject=%5Bwebvr%5D%20YOUR%20TOPIC%20HERE">public-webvr@mozilla.org</a> with subject line “<kbd>[webvr] <i data-lt>… message topic …</i></kbd>” (<a href="https://lists.w3.org/Archives/Public/public-webvr/" rel="discussion">archives</a>)</span>
```

Comment:
Think this was a bug before

Reply:
Indeed, Mailing List is *not* a plural key, so it just takes the last one that was set (which in this case was an empty line).